logging/google/cloud/logging/client.py

```diff
@@ -327,3 +322,21 @@ def setup_logging(self, log_level=logging.INFO,
         handler = self.get_default_handler()
         setup_logging(handler, log_level=log_level,
                       excluded_loggers=excluded_loggers)
+
+    def enable_shutdown_logging(self, thread_dump=True):
+        """Enable shutdown report logging.
+
+        .. note::  This method is only supported on App Engine Flexible.
+
+        This method installs a SIGTERM handler that will report various
+        application metrics to Stackdriver Logging.
```

Comment:
Isn't saying "will report various application metrics" a bit of an overstatement? Don't we just report the full stack?